test/mbta/api_test.exs

```diff
@@ -0,0 +1,111 @@
+defmodule MBTA.ApiTest do
+  use ExUnit.Case, async: false
+
+  alias MBTA.Api
+
+  import Mox
+
+  setup :set_mox_global
+  setup :verify_on_exit!
+
+  describe "get_json/1" do
+    test "normal responses return a JsonApi struct" do
+      expect(HTTPoison.Mock, :get, fn _, _, _ ->
+        {:ok, %HTTPoison.Response{status_code: 200, body: ~s({"data": []})}}
+      end)
```

Comment:
An example of using Mox. This test will hit the underlying `HTTPoison.Mock`. All others will operate a level higher on the `MBTA.Api.Mock`.